zipkin-web/src/main/resources/app/js/page/default.js

```diff
@@ -52,12 +52,15 @@ define(
           const endTs = query.endTs || new Date().getTime();
           const serviceName = query.serviceName || '';
           const annotationQuery = query.annotationQuery || '';
+          const queryWasPerformed = serviceName && serviceName.length > 0;
```

Comment:
This merely replicates the existing behavior :wink:
I agree that we should replace it with traces.length > 0.